packages/fx-core/src/core/error.ts

```diff
@@ -15,6 +15,24 @@ export function ProjectFolderExistError(path: string) {
   );
 }
 
+export function ProjectFolderNotExistError(path: string) {
+  return new UserError(
+    "ProjectFolderNotExistError",
+    `Path ${path} does not exist. Select a different folder.`,
+    CoreSource,
+    new Error().stack
+  );
+}
+
+export function MigrateNotImplementError(path: string) {
+  return new SystemError(
+    "MigrateNotImplement",
```

Comment:
MigrateNotImplement**ed** may be better?

Reply:
Thx, it has been changed.